t/mojolicious/lite_app.t

```diff
@@ -552,6 +558,9 @@ $t->get_ok('/alterformat.html')->status_is(404)->header_is(Server => 'Mojoliciou
 $t->get_ok('/noformat')->status_is(200)->header_is(Server => 'Mojolicious (Perl)')->content_is('xml/noformat');
 $t->get_ok('/noformat.xml')->status_is(404)->header_is(Server => 'Mojolicious (Perl)')->content_is("Oops!\n");
 
+# Echo JSON via query request
+$t->query_ok('/echojson', json => {message => {an => [qw/object of sorts/]}})->json_is('/an' => [qw/object of sorts/]);
```

Comment:
You can also test with another method to make sure it's not just an `any` route.